lib/puppet/face/catalog.rb

```diff
@@ -3,7 +3,7 @@
 require_relative '../../puppet/indirector/face'
 
 Puppet::Indirector::Face.define(:catalog, '0.0.1') do
-  copyright "Puppet Inc.", 2011
+  copyright "Puppet Inc., Vox Pupuli", 2011
```

Comment:
worth pointing out that the Puppet Face API doesn't appear to support multiple copyrights. That's why it's combined into a single string.